src/applications/mhv/landing-page/tests/e2e/restrict-cerner-patients/cerner.patients.cypress.spec.js

```diff
@@ -20,6 +20,7 @@ describe(appName, () => {
       LandingPage.validatePageLoaded();
       LandingPage.validateURL();
       cy.injectAxeThenAxeCheck();
+      cy.findByRole('heading', { name: 'Appointments' }).should.exist;
```

Comment:
I'm wondering if this test file should be removed or renamed. The path name `e2e/restrict-cerner-patients` should change if we decide to keep it.

Reply:
The only reason I left it is that Cerner facilities are different from Vista facilities, but we don't make a check for that in the code. I can remove it.